lib/IRGen/IRGenSIL.cpp

```diff
@@ -1163,8 +1163,42 @@ class IRGenSILFunction :
           emitPackCountDebugVariable(Shape);
         }
       });
+
+    if (auto *BGT = llvm::dyn_cast<BoundGenericType>(Ty)) {
+      auto Decl = BGT->getDecl();
+      auto GE = Decl->getGenericEnvironment();
+      auto Requirements = BGT->getDecl()
+                              ->getGenericEnvironment()
+                              ->getGenericSignature()
```

Comment:
You can replace `->getGenericEnvironment()->getGenericSignature()` with `->getGenericSignature()`